exams/pearson_test.py

```diff
@@ -1,32 +1,35 @@
 """
 Tests for exams.pearson module
 """
-from datetime import datetime
+from datetime import date, datetime
 from unittest import mock
 from unittest.mock import patch
 
 import io
```

Comment:
For me, it's easier to scan imports for where I need to edit/add an import. The style I'm used to, is imports sorted by PEP8, separated by `import` and `from .. import ..` styles. I haven't seen a consistent organization of our imports aside from minimal PEP8 adherence (sometimes I've found imports to be in jumbled order within import blocks), but I'm happy to make adjustments to the style I'm writing in to conform to the project.

Do we have a prescribed import organization method that's more specific than PEP8?